scripts/release/createWebModules.js

```diff
@@ -126,8 +135,41 @@ async function commonActions(moduleInfo, widgets) {
         );
     }
     await commitAndCreatePullRequest(moduleInfo);
-    await updateTestProjectWithWidgetsAndAtlas(moduleInfo, tmpFolder, widgets);
+    await updateTestProjectWithJavascript(tmpFolder, moduleInfo);
+    const mpkOutput = await createMPK(tmpFolder, moduleInfo, regex.excludeFiles);
+
+    await createGithubReleaseFrom({
+        title: `${moduleInfo.nameWithSpace} - Marketplace Release v${moduleInfo.version}`,
+        body: moduleChangelogs.replace(/"/g, "'"),
+        tag: `${moduleFolderNameInRepo}-v${moduleInfo.version}`,
+        mpkOutput,
+        isDraft: true
+    });
+    await execShellCommand(`rm -rf ${tmpFolder}`);
+
+    console.log("Done.");
+}
+
+async function commonActions(moduleInfo, widgets = [], exportWithoutSnippet = false, updateWidgetsChangelogs = false) {
```

Comment:
Nit: `exportWithoutSnippet` is already a negation, maybe we can just use `exportWithSnippet` instead to avoid another negation in the var name already? 😄

Reply:
Done